third_party/README.md

```diff
@@ -0,0 +1,2 @@
+See `./install.sh` for information on where the `*.js` files in this directory
```

Comment:
No, the necessary JS files are included in the repository. ./install.sh
only needs to be run to update to newer versions. (Was this comment
deleted? I couldn't find it on GH.)

On Tue, Sep 20, 2016 at 2:32 PM, Julian Toledo <notifications@github.com>
wrote:

> In third_party/README.md
> <https://github.com/GoogleChrome/gulliver/pull/165#discussion_r79610328>:
>
> > @@ -0,0 +1,2 @@
> > +See `./install.sh` for information on where the `*.js` files in this directory
>
> @ithinkihaveacat <https://github.com/ithinkihaveacat> Do we need to run
> ./install.sh to be able to use it?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleChrome/gulliver/pull/165/files/1ae07a7d45a999306997776b633399eb04091abd#r79610328>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AADILEGmssP64LrBzqkrDEK8kUoDNTHDks5qr-B6gaJpZM4KBjT9>
> .
>